.changeset/fresh-poems-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+"@equinor/react-native-mad-tag-ocr": patch
+---
+
+Update `OCRCamera` component to prevent build error
```

Comment:
To me it doesn't look like you updated react-native-mad-tag-ocr at all. Maybe this changeset should be removed?